npm/wizer/wizer

```diff
@@ -0,0 +1 @@
+placeholder file for the wizer binary to be copied to
```

Comment:
nitpick: missing a newline at the end of the file

Reply:
I should set up my editor to add an empty newline at the end of every file 😅